### PR TITLE
#4624 [Manager] Create workspace restart indicator

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4624 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Workspace/Settings/SettingsHooks.js [hunk 1]
```diff
@@ -161,6 +170,7 @@ export const useConfig = ({ openDeleteProgressDialog, workspace }) => {
         //   type: SETTINGS_COMPONENT_TYPES.PAGE,
         // },
         {
+          badge: { count: countOfZookeeperNodesChanged },
```
Comment (eechih) [inline ohara-manager/client/src/components/Workspace/Settings/SettingsHooks.js:173]: @konekoya 這個設定是用於顯示 MuiBadge，後續再麻煩錦現負責 UI 的部分，謝謝。

File: ohara-manager/client/src/components/Workspace/Settings/SettingsMain.js [hunk 2]
```diff
@@ -98,24 +109,30 @@ const SettingsMain = ({
...
-                            </IconButton>
-                          </ListItemSecondaryAction>
-                        )}
-
-                        {type === SETTINGS_COMPONENT_TYPES.CUSTOMIZED && (
-                          <>{children}</>
-                        )}
+                        <Badge
+                          component="div"
+                          badgeContent={badge?.count}
```
Comment (konekoya) [inline ohara-manager/client/src/components/Workspace/Settings/SettingsMain.js:114]: The value 0 is not displaying by default: https://material-ui.com/components/badges/#badge-visibility

File: ohara-manager/client/src/components/Workspace/Settings/SettingsHooks.js (hunk 1 above)
Comment (konekoya) [reply ohara-manager/client/src/components/Workspace/Settings/SettingsHooks.js]: I've added the badge icon. Please take a look and let me know if there anything that i can be of any help :)

File: ohara-manager/client/src/components/Workspace/Settings/SettingsMain.js (hunk 2 above)
Comment (eechih) [reply ohara-manager/client/src/components/Workspace/Settings/SettingsMain.js:114]: Got it

File: ohara-manager/client/src/components/Workspace/Settings/SettingsHooks.js (hunk 1 above)
Comment (eechih) [reply ohara-manager/client/src/components/Workspace/Settings/SettingsHooks.js]: @konekoya Thanks for the help :)

File: ohara-manager/client/src/components/Workspace/Settings/SettingsMain.js [hunk 3]
```diff
@@ -98,24 +109,30 @@ const SettingsMain = ({
...
+                          color="primary"
```
Comment (eechih) [inline ohara-manager/client/src/components/Workspace/Settings/SettingsMain.js:115]: @konekoya Can we change the Badge color to `theme.palette.warning.main`?

Comment (konekoya) [reply ohara-manager/client/src/components/Workspace/Settings/SettingsMain.js:115]: Absolutely 👌

Comment (konekoya) [reply ohara-manager/client/src/components/Workspace/Settings/SettingsMain.js]: Done. I also change the text color from black to white :)

Comment (eechih) [reply ohara-manager/client/src/components/Workspace/Settings/SettingsMain.js]: 👍 Thanks.

**Pull request merged**